README.md

```diff
@@ -102,7 +102,7 @@ and others would not be possible without your help.
 ## Online Documentation
 
 Since Kyuubi 1.3.0-incubating, the Kyuubi online documentation is hosted by [https://kyuubi.apache.org/](https://kyuubi.apache.org/).
-You can find the latest Kyuubi documentation on [this web page](https://kyuubi.apache.org/docs/latest/).
+You can find the latest Kyuubi documentation on [this web page](https://kyuubi.apache.org/).
```

Comment:
How about pointing this to `https://kyuubi.readthedocs.io/en/master/`? cc @yaooqinn

Reply:
LGTM

Reply:
@runzhliu would you please update it?

Reply:
this [link](https://kyuubi.readthedocs.io/en/master/) also seems to be unavailable, and always be redirected to https://kyuubi.apache.org/en/master/ @pan3793

Reply:
we are investigating the issue

Reply:
fixed